### ISSUE TITLE
docker-machine: v0.8.0-rc9:  console switch ubuntu: Error running SSH command: netstat -tln

### ISSUE THREAD

**Issue by jujes** [rancher/os#1584]
**RancherOS Version: (ros os version)**
v0.8.0-rc9

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
docker-machine

**sudo ros console switch ubuntu**

```
$ docker-machine start www-py
Starting "www-py"...
(www-py) Check network to re-create if needed...
(www-py) Waiting for an IP...
kMachine "www-py" was started.
Waiting for SSH to be available...
Detecting the provisioner...
Warning: got an invalid line error parsing /etc/os-release: Expected ID_LIKE===========debian to split by '=' char into two strings, instead got 12 strings
Error running SSH command: ssh command error:
command : netstat -tln
err     : exit status 127
output  : bash: netstat: command not found
Error running SSH command: ssh command error:
command : netstat -tln
err     : exit status 127
output  : bash: netstat: command not found
```

---

```
docker@www-py:~$ sudo ros os upgrade -i rancher/os:v0.8.0-rc9
Upgrading to rancher/os:v0.8.0-rc9
Continue [y/N]: y
Pulling os-upgrade (rancher/os:v0.8.0-rc9)...
v0.8.0-rc9: Pulling from rancher/os
0a8490d0dfd3: Pull complete 
54ec58ca6e0b: Pull complete 
c9745547f6d2: Pull complete 
23a9783dbec2: Pull complete 
f495dbccaea7: Pull complete 
f4af75515eb5: Pull complete 
43a64476a514: Pull complete 
bc245a1a8938: Pull complete 
9a11d4548584: Pull complete 
Digest: sha256:5f37cec3a2ea77311a7934b84f4ada93620d4404c3a5a9cbcc131d569c1d1193
Status: Downloaded newer image for rancher/os:v0.8.0-rc9
os-upgrade_1 | > time="2017-02-07T11:30:07Z" level=warning msg="Cloud-config not provided: you might need to provide cloud-config on bootDir with ssh_authorized_keys" 
os-upgrade_1 | Installing from :v0.8.0-rc9
os-upgrade_1 | > time="2017-02-07T11:30:07Z" level=info msg="mountdevice 1, raw false"
os-upgrade_1 | lsblk: 1: not a block device
os-upgrade_1 | > time="2017-02-07T11:30:07Z" level=error msg="error layDownOS exit status 32" 
os-upgrade_1 | > time="2017-02-07T11:30:07Z" level=fatal msg="Failed to run install" err="exit status 32" 
Continue with reboot [y/N]: y

```

**Comment by SvenDowideit** [comment]
at this point, docker-machine boots from ISO - so installing and upgrading like this can't work. I would like RancherOS to detect this and tell the user that it won't work - see #1552

that `os-release` error looks interesting though.

**Comment by SvenDowideit** [comment]
@jujes what vm tech are you using in docker-machine?  virtualbox?

**Comment by SvenDowideit** [comment]
interesting - in the final v0.8.0, the os-release issue didn't happen in my test - the netstat one does

but once the vm is up, I can `docker-machine ssh` into it

```

Error running SSH command: Something went wrong running an SSH command!
command : netstat -tln
err     : exit status 127
output  : bash: netstat: command not found


Unable to verify the Docker daemon is listening: Maximum number of retries (10) exceeded

svend@x260 MINGW64 ~
$ docker-machine ssh test
Welcome to Ubuntu 16.10 (GNU/Linux 4.9.9-rancher x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
Last login: Wed Feb 22 04:08:03 2017 from 10.0.2.2
To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.

docker@test:~$ logout

svend@x260 MINGW64 ~
$ docker-machine ls
NAME   ACTIVE   DRIVER       STATE     URL                         SWARM   DOCKER    ERRORS
test   -        virtualbox   Running   tcp://192.168.99.100:2376           v1.12.6
```

(my current plan it to add netstat to my list of things a console needs :/ )

**Comment by SvenDowideit** [comment]
The netstat issue with ubuntu and debian consoles and docker-machine should now be fixed - I've updated the v0.8.0 images for it.

I'm pretty sure that the os-release file issue was fixed during development, and the lack of error bailout when upgrading a docker-machine is something we'll look at later :)
